ably-ios/ARTClientOptions.h

```diff
@@ -14,24 +14,32 @@ ART_ASSUME_NONNULL_BEGIN
 
 @interface ARTClientOptions : ARTAuthOptions
 
-@property (readonly, getter=getRestHost) NSString *restHost;
-@property (readonly, getter=getRealtimeHost) NSString *realtimeHost;
+@property (readwrite, strong, nonatomic, getter=getRestHost) NSString *restHost;
```

Comment:
I understand the reason why you changed the property to readwrite but I noticed the property has no setter.